charge.go

```diff
@@ -485,7 +485,7 @@ type Charge struct {
 	OnBehalfOf                *Account                    `json:"on_behalf_of"`
 	Outcome                   *ChargeOutcome              `json:"outcome"`
 	Paid                      bool                        `json:"paid"`
-	PaymentIntent             string                      `json:"payment_intent"`
+	PaymentIntent             *PaymentIntent              `json:"payment_intent"`
```

Comment:
This is being approved right now and since it's a breaking change I'm putting this in right now